src/main/java/racingcar/RacingCarGame.java

```diff
@@ -0,0 +1,39 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Map;
+import racingcar.controller.InputController;
+import racingcar.controller.OutputController;
+import racingcar.domain.InputContainer;
+import racingcar.service.RacingCarService;
+import racingcar.validator.Validator;
+
+public class RacingCarGame {
+    Validator validator = new Validator();
+    InputController inputController = new InputController();
+    OutputController outputController = new OutputController();
+    RacingCarService racingCarService = new RacingCarService();
+
+    public void start() {
+        InputContainer inputContainer = inputController.readConsole();
+        int attempts = inputContainer.getAttempts();
+        validator.attemptValidate(attempts);
+
+        String[] cars = inputContainer.getCars().split(",");
+        Map<String, String> carMoveMap = setupCars(cars);
```

Comment:
제 생각에는 Map의 Key와 Value에 해당하는 부분을 String 보다는 의미가 있는 객체로 나타내 주시거나, Map 자체를 객체로 변환해서 사용하라는 것으로 이해 했습니다. 

```suggestion
        Map<CarName, Position> carMoveMap = setupCars(cars);
```

```suggestion
        List<Car> cars = setupCars(cars);
```